### DISCUSSION TITLE
Air flow disabled by default

### DISCUSSION THREAD

**Discussion by blauret** [michaelarnauts/home-assistant-comfoconnect#97] [Q&A] [answered]
First, it's my first posting in the HA comfoconnect community. Thanks for the work here. It's pretty amazing!

Is there a particular reason to disable it by default? You can see it in the Zehnder app and it's nice to monitor the current fan setting.

```
        native_unit_of_measurement=UnitOfVolumeFlowRate.CUBIC_METERS_PER_HOUR,
        icon="mdi:fan-plus",
        ccb_sensor=SENSORS.get(SENSOR_FAN_SUPPLY_FLOW),
        entity_registry_enabled_default=False,
```

**Comment by michaelarnauts** [comment, marked as answer]
It's disabled by default because I've had a few cases where the Comfoconnect LAN C (on some firmware versions) could freeze due to too many messages. The air flow sensors are constantly fluctuating, causing quite a bit of updates.

It's therefore disabled to use a minimal default. Feel free to enable them from the GUI.

Note that there is no filtering and updates are pushed every second of so. This might cause the Home Assistant database to grow in size unexpectedly.

**Reply by blauret** [reply]
Thanks for the answer. All makes sense. Much appreciated.
